Sources/util.swift

```diff
@@ -27,3 +27,40 @@ final class AssociatedObject<T: Any> {
 		}
 	}
 }
+
+class LifetimeAssociation {
+	private(set) weak var object: AnyObject?
+	private(set) weak var owner: AnyObject?
```

Comment:
Why are these `private(set)` and not `private`? I don't see them being read anywhere outside the class.